libass/ass_bitmap.c

```diff
@@ -209,9 +214,21 @@ void fix_outline(Bitmap *bm_g, Bitmap *bm_o)
     uint8_t *g = bm_g->buffer + (t - bm_g->top) * bm_g->stride + (l - bm_g->left);
     uint8_t *o = bm_o->buffer + (t - bm_o->top) * bm_o->stride + (l - bm_o->left);
 
+    // Use a number just above 65025 to ensure den > 0 and we don't divide
+    // by zero. This allows us to save a branch and assist autovectorization.
+    // The cost (which we accept) with IEEE float32 is that just two out
+    // of the 33 million possible operand tuples round in the "wrong"
+    // direction after evaluating to almost exactly an integer and a half.
+    const float over65025 = nextafterf(65025, FLT_MAX);
```

Comment:
I ultimately decided to just go with `nextafterf` as the universally safest choice (with no assumptions about the floating point format). The drawbacks I considered are:
* It may be absent on some older systems. But we make extensive use of `lrint` and I invoke `lround` once in karaoke code, and I’d like to hope that libraries that contain those functions also have `nextafterf`.
* It’s not necessarily evaluated as a compile-time constant. But I’ve hoisted it out of the pixel-wise loop, so it’s now a single call per bitmap, which shouldn’t hurt.

I tried to emulate this particular `nextafterf` call through basic arithmetic, but couldn’t make it work without assuming things about the floating point format. Which is likely very safe in practice, but if I can avoid it at no significant cost, I’d rather avoid it.